tests/e2e/tidbcluster/stability.go

```diff
@@ -1030,17 +1031,17 @@ var _ = ginkgo.Describe("[tidb-operator][Stability]", func() {
 			tc.Spec.TiDB.Replicas = 1
 			tc.Spec.TiKV.Replicas = 3
 			err := genericCli.Create(context.TODO(), tc)
-			framework.ExpectNoError(err)
+			framework.ExpectNoError(err, "failed to create TidbCluster: %v", tc)
```

Comment:
Is this intended to print the whole tc?

Reply:
yes, we can refine all the `explain` parameters in e2e in the long term.